common/app/model/content.scala

```diff
@@ -353,8 +351,6 @@ final case class Content(
   val quizzes: Seq[Quiz] = atoms.map(_.quizzes).getOrElse(Nil)
   val media: Seq[MediaAtom] = atoms.map(_.media).getOrElse(Nil)
 
-  val nonCompliantOutbrainAmp: Boolean = (hasStoryPackage && tags.series.nonEmpty) || (tags.series.length > 1)
```

Comment:
Because hasStoryPackage is always set to false, this reduces down to `tags.series.length > 1`.